<?xml version="1.0"?>

<launch>

    <node pkg="data_analysis" type="nmea_extraction.py" name="nmea_exraction" output="screen">
        <param name="file_save_path" value=""/> <!-- default_value="$(WORK_SPACE)/src/data_analysis/doc" -->
        <param name="nmea_topic" value=""/> <!-- default_value="nmea_sentence" -->
    </node>

    <node pkg="data_analysis" type="navsat_fix_extraction.py" name="navsat_fix_extraction" output="screen">
        <param name="fix_topic" value=""/> <!-- default_value="fix" -->
        <param name="fix_filtered_topic" value=""/> <!-- default_value="gps/filtered" -->
        <param name="file_save_path" value=""/> <!-- default_value="$(WORK_SPACE)/src/data_analysis/doc" -->
    </node>

</launch>